db/migrate/20200318165103_remove_zip_from_shelters.rb

```diff
@@ -0,0 +1,4 @@
+class RemoveZipFromShelters < ActiveRecord::Migration[5.1]
+  def change
```

Comment:
I was just practicing db migration, i was having a little trouble with migration commands in terminal